add PoE documentation (#70)
bisdn/bisdn.github.io: platform_configuration/power_over_ethernet.md

---
title: Power Over Ethernet (PoE)
parent: Platform Configuration
nav_order: 5
---

**WARNING**: PoE is currently an experimental feature.
{: .label .label-red }

# Power Over Ethernet (PoE)

## Introduction

> Power over Ethernet (PoE) is a technique that allows providing electric power and data over an Ethernet cable. The original PoE standard (IEEE 802.3af) supplies up to 15.4 W per port, and was increased to 25.5 W with a revision to the standard (802.3at).

Reviewer: ```suggestion
Power over Ethernet (PoE) is a technique that allows providing electric power and data over an ethernet cable. The original PoE standard (IEEE 802.3af) supplied up to 15.4 W per port, but the revision of the standard (802.3at) increased it to 25.5 W.
```

> Two types of devices supporting PoE are considered regarding PoE capabilities: the *Power Sourcing Equipment (PSE)* which are the devices capable of providing power on the cable; and the *Powered Device (PD)* are devices that consume energy provided by PoE. Some BISDN Linux supported switches can act as a Power Sourcing Equipment.  

Reviewer: ```suggestion
Two types of devices supporting PoE are considered regarding PoE capabilities: the *Power Sourcing Equipment (PSE)*, which are the devices capable of providing power on the cable; and the *Powered Device (PD)*, which are devices that consume energy provided by PoE. Some BISDN Linux supported switches can act as a Power Sourcing Equipment (please refer to the platform specification provided by your vendor).  
```

## Configuration

> BISDN Linux comes with an application `poectl` that allows for PoE configuration.

Reviewer: ```suggestion
BISDN Linux comes with the command line utility `poectl`, which provides a simple and convenient way to configure PoE.
```

```
:~$ poectl -h
poectl for enable, disable, status and measurements of PoE ports
 
 Usage:  poectl [OPTIONS] PORT | all
 
 OPTIONS:
  -h   help
  -v   version
  -e   enable PoE
  -d   disable PoE
  -s   PoE status
  -mV  PoE voltage
  -mC  PoE current
  -mT  PoE temperature
  -mP  PoE power
```

> Usage of the poectl application is done by running the following command:

Reviewer: what are you trying to say here?

```
:~$ poectl [OPTIONS] PORT | all
```

> To enable PoE on a port follow the following command:

Reviewer: ```suggestion
To enable PoE on a port run the following command:
```

Reviewer: i think we probably want to start with an example that shows how to enable it on all ports
